izadji/src/components/ProjectSlide/ProjectSlide.tsx

```diff
@@ -30,9 +35,12 @@ const ProjectSlide = ({
           <span className={styles.hero__slider__subtitle}>{date}</span>
           <h2 className={styles.hero__slider__title}>{title}</h2>
           <p className={styles.hero__slider__text}>{description}</p>
-          <Link to={link} className={styles.btn}>
+          <a
```

Comment:
Please, use react router Link. Also, suggestion would be to use **link** prop that is defined, so that you don't need to pass **id** prop into this component. Create link prop on component mapping.

Reply:
Okay. I will wait for your pr with the necessary changes and then this pr will be completed.